sdk/core/core-client/review/core-client.api.md

```diff
@@ -297,6 +291,9 @@ export interface PolymorphicDiscriminator {
 // @public
 export type QueryCollectionFormat = "CSV" | "SSV" | "TSV" | "Pipes" | "Multi";
 
+// @public
+export type RawResponseCallback = (rawResponse: FullOperationResponse, flatResponse: unknown) => void;
```

Comment:
Should we make the second argument optional?

Reply:
Optional here would mean the caller doesn't need to pass it, but we should always have something to pass along, right? 

The implementer of the callback doesn't need to receive the parameter or do anything with it, e.g. this works

```ts
doFoo({onResponse: (rawResponse) => {
    console.log(rawResponse);
}});
```